File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/trials/I1/go/trial_01/solution.go
package main

import "fmt"
import "strconv"

func main () {
	var state int64 = 7
	var total int64 = 0
	var biggest int64 = 0
	var evenCount int64 = 0
	var joined string = ""
	var i int64 = 0
	for i = 0; i < 50; i = i + 1 {
		state = (state * 48271) % 2147483647
		var term int64 = state % 1000
		total = total + term
		if term > biggest {
			biggest = term
		}
		if term % 2 == 0 {
			evenCount = evenCount + 1
		}
		if i < 5 {
			if i > 0 {
				joined = joined + "-"
			}
			joined = joined + strconv.FormatInt(term, 10)
		}
	}
	fmt.Println("SUM " + strconv.FormatInt(total, 10))
	fmt.Println("MAX " + strconv.FormatInt(biggest, 10))
	fmt.Println("EVENS " + strconv.FormatInt(evenCount, 10))
	fmt.Println("JOINED " + joined)
}

File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/trials/I1/typescript/trial_01/solution.js
"use strict";
function collect(count) {
    const out = [];
    let state = 7;
    for (let step = 0; step < count; step = step + 1) {
        state = (state * 48271) % 2147483647;
        out.push(state % 1000);
    }
    return out;
}
function bigger(left, right) {
    if (left > right) {
        return left;
    }
    else {
        return right;
    }
}
function main() {
    const items = collect(50);
    let total = 0;
    let biggest = items[0];
    let evens = 0;
    for (const item of items) {
        total = total + item;
        biggest = bigger(biggest, item);
        if (item % 2 === 0) {
            evens = evens + 1;
        }
    }
    const pieces = [];
    for (let index = 0; index < 5; index = index + 1) {
        pieces.push(String(items[index]));
    }
    const joined = pieces.join("-");
    console.log("SUM " + String(total));
    console.log("MAX " + String(biggest));
    console.log("EVENS " + String(evens));
    console.log("JOINED " + joined);
}
main();
